content/en/docs/v3.5/op-guide/kubernetes.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Run etcd clusters as a Kubernetes StatefulSet
+weight: 4200
+description: Running etcd as a Kubernetes StatefulSet
+---
+
+Below demonstrates how to perform the [static bootstrap process](../clustering/#static) as a Kubernetes StatefulSet.
+
+## Example Manifest
+This manifest contains a service and statefulset for deploying a static etcd cluster in kubernetes.
+
+If you copy the contents of the manifest into a file named `etcd.yaml`, it can be applied to a cluster with this command.
```

Comment:
Suggestion to mention that later in the document (when enabling TLS), it is expected to have them applied in the default namespace.

Reply:
It's explicitly expressed in the examples:
```
metadata:
  name: <whatever the thing is>
  namespace: default
```

Reply:
Thanks for the clarification.